### PR TITLE
Feature my app

### PR DESCRIPTION
**Pull request by iambenkis**
# BookShelves metrics app - Metrics webapp

## Project Description :book: 
In this React capstone project  I built a mobile web application to check a list of metrics (numeric values)  using React and Redux. 

I selected an API that provides numeric data about books and I build the webapp around that. The web contains two pages :

- one page with a list of books which is the homepage.
- one page for the book's details; .
*IMPORTANT NOTE: Read **all** requirements before you start building your project.*

### Checklists :100: 

- There are [no linter errors](https://github.com/microverseinc/linters-config). :v: 
- I used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). :v: 
- I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). :v: 
- I apply [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html- css/blob/main/articles/html_css_best_practices.md). :v: 
- I apply [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). :v: 
 - I use an API [BOOK_SELVES_API](https://rapidapi.com/Ionut-Cora/api/bookshelves/) :v: 
- I performed test in this project using jest :v:

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      className="bg"
+    />
+    <div className="band">
+      <p>BOOK WITH STARS AND PRICE</p>
+      <div>
+        <input type="text" placeholder="Search..." />
+        <div className="div-search">
+          <BiSearchAlt2 className="search" />
+        </div>
+      </div>
```
Comment (zieeco) [inline src/components/Home/Home.js]: - Hey, great job with the project, I admire your effort so far, You've done well. I just wanted to point out something concerning the search. As it's a requirement to add a search functionality so a user could search by a book name, I don't seem to see it working as intended. when a user searches for a book, it should show up what matches the search e.g, the name of the book, but on the contrary, it doesn't work that way. Please, try to add this implementation to meet this part of the project requirement. :+1:

File: README.md [hunk 2]
```diff
@@ -1,70 +1,116 @@
...
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[Live Demo](https://634a5de47268306b6cf8fb28--scintillating-cupcake-d003b8.netlify.app/)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Project Video Description
 
-### `npm run eject`
+[Video Presentation](https://drive.google.com/file/d/1xD4SuzL1vCv8R33JCTENYaMD6WgHxqoh/view?usp=sharing)
```
Comment (zieeco) [inline README.md]: - [ ]  Please, let's try to keep in mind to be attentive to details. it's a skill that would make you stand out in your future job. The video is supposed to have a maximum length of 5min. It is best to use [loom](https://www.loom.com/) as this helps you not to record more than 5mins. please kindly update the video so it meets the project requirements.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @iambenkis,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @iambenkis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**